code/game/atoms_movable.dm

```diff
@@ -50,6 +50,10 @@
 	// When this object moves. (args: loc)
 	var/event/on_moved
 
+	var/atom/movable/tether_master
+	var/list/tether_slaves = list()
```

Comment:
Lazily initialize these lists, it's 2 lists per movable atom and it causes quite a problem.

Reply:
You mean, initialize them only when a tether is actually added to them?

Reply:
Yes, that is what I mean.

Lazily initializing something means not creating something until it's needed, because it'd just waste memory/CPU time.